mllib/src/test/scala/org/apache/spark/mllib/rdd/MLPairRDDFunctionsSuite.scala

```diff
@@ -33,4 +33,16 @@ class MLPairRDDFunctionsSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(topMap(3) === Array(7, 5, 2))
     assert(topMap(5) === Array(1))
   }
+
+  test("topByKeyUsingBuckets") {
```

Comment:
yes it's supposed to be the same. unfortunately I can't see a way to verify the number of reducers being actually used. it's not a correctness issue though, so I think verifying manually should be enough ---- I did verify on spark UI that it's invoking the intended reducer count